### DISCUSSION TITLE
Nextjs10 not working in IE11 fine with all my arrow functions

### DISCUSSION THREAD

**Discussion by anialamo** [vercel/next.js#20231] [Help] [answered]
# Bug report
Hi! I have a Nextjs10 webapp. This works fine in Chrome, Safari, Mozilla, but in IE11 the app doesnt work fine. All my arrow functions dont work. How can I get my application to work fine in IE 11?

## Describe the bug

In my IE11 browser this section doesn't do anything (in Chrome its open, in IE11 do nothing, its an arrow function):
![image](https://user-images.githubusercontent.com/35197781/102390540-d12dfa00-3fd4-11eb-9f01-7952ee4fd66c.png)

My next.config.js is
```
const withSass = require("@zeit/next-sass");

const withTM = require('next-transpile-modules')([]); // pass the modules you would like to see transpiled, I only need my own code

const localeLanguages = [
     { label: 'en', name: 'English' },
     { label: 'es', name: 'Español' }     
];

// Extend your Next config for advanced behavior
// See https://nextjs.org/docs/api-reference/next.config.js/introduction
let nextConfig = {
     
     publicRuntimeConfig: {
          localeLanguages
     },

     i18n: {
          // These are all the locales you want to support in
          // your application
          locales: ['en', 'es'],
          // This is the default locale you want to be used when visiting
          // a non-locale prefixed path e.g. `/hello`
          defaultLocale: 'en',
          localeDetection: true,
     },     
};
// Add the Next SASS plugin
nextConfig = withSass(withTM(nextConfig));

module.exports = nextConfig;


```

## System information

- OS: Windows 10
- Browser (if applies) IE11
- Version of Next.js: 10.0.3

**Comment by leerob** [comment, marked as answer]
Hello, we would need a reproducible example to investigate further (e.g. codesandbox). Ideally, one that doesn't use external community packages like `next-transpile-modules`. If you can do this, we can investigate further. 

Thank you!
